src/servicemgr.cpp

```diff
@@ -526,13 +527,11 @@ namespace koi {
 				case Svc_Fail: {
 					if (s._state == Svc_Starting) {
 						LOG_WARN("Overriding starting transition to stop immediately: %s", s._name.c_str());
-						s._running.kill(SIGKILL);
+						s._running.termkill(TERMINATE_TIMEOUT);
 						s.unrun();
-						if (s._state == Svc_Starting) {
-							s.transition(Svc_Started);
-							LOG_WARN("Forcing stop");
-							s.stop(_logproxy.inpipe);
-						}
+						s.transition(Svc_Started);
+						LOG_WARN("Forcing stop");
+						s.stop(_logproxy.inpipe);
```

Comment:
What is the reason for this change? I don't remember this part from the earlier diff..

Reply:
This change was there earlier too.
The condition check `if (s._state == Svc_Starting)` is duplicated.
Also `s._state` didn't change in between.
So removed this redundant check.

Reply:
Ah yes, I see now. Thanks!